solutions/remotemonitoring/single-vm/docker-compose.dotnet.yml

```diff
@@ -85,7 +85,6 @@ services:
       - default_net
     depends_on:
       - auth
-      - diagnostics
```

Comment:
why is this not needed?

Reply:
It introduced loop dependency:  telemetry->diagnostics->config->telemetry.